### PR TITLE
Seed the gatekeeper-operator FBC

### PR DESCRIPTION
**Pull request by dhaiducek**
ref: https://issues.redhat.com/browse/ACM-15393

### CODE REVIEW

**Comment by dhaiducek** [comment]
Merging blocked by:
- https://github.com/openshift/release/pull/60019

File: catalog.Dockerfile.rhtap [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# The builder image is expected to contain /bin/opm (with serve subcommand)
+FROM registry.redhat.io/openshift4/ose-operator-registry-rhel9:v4.17 as builder
```
Comment (yiraeChristineKim) [inline catalog.Dockerfile.rhtap:2]: Should we update this whenever a new OpenShift release comes out?

File: catalog/bundles/bundle-v3.17.0.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
...eyJ0eXBlIjoiaW50ZWdlciJ9LHsidHlwZSI6InN0cmluZyJ9XSwicGF0dGVybiI6Il4oXFwrfC0pPygoWzAtOV0rKFxcLlswLTldKik/KXwoXFwuWzAtOV0rKSkoKFtLTUdUUEVdaSl8W251bWtNR1RQRV18KFtlRV0oXFwrfC0pPygoWzAtOV0rKFxcLlswLTldKik/KXwoXFwuWzAtOV0rKSkpKT8kIiwieC1rdWJlcm5ldGVzLWludC1vci1zdHJpbmciOnRydWV9LCJkZXNjcmlwdGlvbiI6IlJlcXVlc3RzIGRlc2NyaWJlcyB0aGUgbWluaW11bSBhbW91bnQgb2YgY29tcHV0ZSByZXNvdXJjZXMgcmVxdWlyZWQuXG5JZiBSZXF1ZXN0cyBpcyBvbWl0dGVkIGZvciBhIGNvbnRhaW5lciwgaXQgZGVmYXVsdHMgdG8gTGltaXRzIGlmIHRoYXQgaXMgZXhwbGljaXRseSBzcGVjaWZpZWQsXG5vdGhlcndpc2UgdG8gYW4gaW1wbGVtZW50YXRpb24tZGVmaW5lZCB2YWx1ZS4gUmVxdWVzdHMgY2Fubm90IGV4Y2VlZCBMaW1pdHMuXG5Nb3JlIGluZm86IGh0dHBzOi8va3ViZXJuZXRlcy5pby9kb2NzL2NvbmNlcHRzL2NvbmZpZ3VyYXRpb24vbWFuYWdlLXJlc291cmNlcy1jb250YWluZXJzLyIsInR5cGUiOiJvYmplY3QifX0sInR5cGUiOiJvYmplY3QifX0sInR5cGUiOiJvYmplY3QifX0sInR5cGUiOiJvYmplY3QifSwic3RhdHVzIjp7ImRlc2NyaXB0aW9uIjoiR2F0ZWtlZXBlclN0YXR1cyBkZWZpbmVzIHRoZSBvYnNlcnZlZCBzdGF0ZSBvZiBHYXRla2VlcGVyIiwidHlwZSI6Im9iamVjdCJ9fSwidHlwZSI6Im9iamVjdCJ9fSwic2VydmVkIjp0cnVlLCJzdG9yYWdlIjp0cnVlLCJzdWJyZXNvdXJjZXMiOnsic3RhdHVzIjp7fX19XX0sInN0YXR1cyI6eyJhY2NlcHRlZE5hbWVzIjp7ImtpbmQiOiIiLCJwbHVyYWwiOiIifSwiY29uZGl0aW9ucyI6bnVsbCwic3RvcmVkVmVyc2lvbnMiOm51bGx9fQ==
```
Comment (yiraeChristineKim) [inline catalog/bundles/bundle-v3.17.0.yaml:17]: What is this data field for?

**Review by yiraeChristineKim — APPROVED** [review]

File: catalog/bundles/bundle-v3.17.0.yaml (hunk 2 above)
Comment (dhaiducek) [reply catalog/bundles/bundle-v3.17.0.yaml:17]: Each one is the operator bundle YAML (operator CRD, CSV, RBAC)

**Pull request merged**